### PR TITLE
Upgrade UserSecurityReport to SS4

### PR DESCRIPTION
**Pull request by Firesphere**
Removed subsites because it's incompatible with SS4

### CODE REVIEW

File: _config/subsite.yml [hunk 1]
```diff
@@ -6,6 +6,6 @@ Only:
 UserSecurityReport:
   extensions:
```
Comment (flamerohr) [inline _config/subsite.yml]: I wonder if there's a way to ignore this yaml... if nothing else, comment it all out?

File: src/subsites/SubsiteMemberReportExtension.php [hunk 2]
```diff
@@ -1,8 +1,13 @@
 <?php
+namespace SilverStripe\SecurityReport;
```
Comment (flamerohr) [inline src/subsites/SubsiteMemberReportExtension.php]: perhaps add a `return;` at the top here to stop the class from being loaded at all?
Better than loading this unsupported class, and cleaner than commenting out the whole file

File: src/subsites/SubsiteSecurityReport.php [hunk 3]
```diff
@@ -1,4 +1,7 @@
 <?php
+namespace SilverStripe\SecurityReport;
```
Comment (flamerohr) [inline src/subsites/SubsiteSecurityReport.php]: Same here, perhaps add a `return;` to the top to preventing loading

File: tests/SubsitesReportTest.php [hunk 4]
```diff
@@ -1,31 +1,42 @@
 <?php
 /**
+ * @broken, SubSites currently is unavailable on SS4
```
Comment (flamerohr) [inline tests/SubsitesReportTest.php]: Same here, add a `return;` to the top, rather than commenting out the whole file

Comment (Firesphere) [inline tests/SubsitesReportTest.php]: Yep, good idea. I don't want to remove it, for  possible future workings.

File: src/subsites/SubsiteSecurityReport.php (hunk 3 above)
Comment (Firesphere) [inline src/subsites/SubsiteSecurityReport.php]: Tests seem to ignore return or mark skipped. So leaving it with comment for now.

The report won't do anything with the YML commented out, so shouldn't be to big an issue?

Comment (flamerohr) [inline src/subsites/SubsiteSecurityReport.php]: Oh ok, yeah, leaving the yaml commented should be fine :)

**Pull request merged**